.github/workflows/plugin-sdk-version-check.yml

```diff
@@ -0,0 +1,48 @@
+name: Plugin SDK - Match version from html5 package with bigbluebutton.properties
+on:
+  push:
+    branches:
+      - "develop"
+      - "v2.[5-9].x-release"
+      - "v[3-9].*.x-release"
+  pull_request:
+    types: [opened, synchronize, reopened]
+permissions:
+  contents: read
+jobs:
+  plugin-sdk-version-check:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 1
+          
+      - name: Ensure jq is installed
+        run: sudo apt-get update && sudo apt-get install -y jq
+
+      - name: Merge branches
+        uses: ./.github/actions/merge-branches
+
+      - name: Extract plugin-sdk version from html5 
+        working-directory: bigbluebutton-html5
+        run: |
+          VERSION=$(jq -r '.dependencies["bigbluebutton-html-plugin-sdk"]' package.json | sed 's/^[^0-9]*//')
+          echo "SDK version from package.json: $VERSION"
+          echo "HTML_SDK_VERSION=$VERSION" >> $GITHUB_ENV
+
+      - name: Extract plugin-sdk version from bigbluebutton.properties 
+        working-directory: bigbluebutton-web
+        run: |
+          VERSION=$(grep '^htmlPluginSdkVersion=' grails-app/conf/bigbluebutton.properties | cut -d'=' -f2 | tr -d '[:space:]')
+          echo "SDK version from bigbluebutton.properties: $VERSION"
+          echo "PROPERTIES_SDK_VERSION=$VERSION" >> $GITHUB_ENV
+
+      - name: Check plugin-sdk version of html5 against bigbluebutton.properties
+        run: |
+          echo "Comparing versions..."
+          if [ "$HTML_SDK_VERSION" != "$PROPERTIES_SDK_VERSION" ]; then
```

Comment:
I'm wondering what will happen in the case of a test commit that uses `https://codeload`.

It should be handled, but I recommend making this action fail intentionally. That way, we prevent someone from accidentally merging the PR while it's still relying on a temporary package.